proof/invariant-abstract/AARCH64/ArchFinalise_AI.thy

```diff
@@ -1900,9 +1898,7 @@ lemma (* dmo_replaceable_or_arch_update *) [Finalise_AI_assms,wp]:
 
 end
 
-context begin interpretation Arch .
-requalify_consts replaceable_or_arch_update
-end
+arch_requalify_consts replaceable_or_arch_update
```

Comment:
Requalification in an Arch file

Reply:
Unavoidable: const introduced in Arch locale that needs to be referenced generically in this theory.